### DISCUSSION TITLE
Language Selection on Startup

### DISCUSSION THREAD

**Discussion by builderjer** [OpenVoiceOS/ovos-buildroot#163] [Ideas]
I moved this to here because it is not specific to the raspbian-ovos image

https://github.com/orgs/OpenVoiceOS/discussions/160#discussioncomment-5502151

It was suggested that language selection should be configurable before any other setup.  Not sure how this will or could work, but I think the discussion will get more action here.

**Comment by emphasize** [comment]
At some point this was on the/a roadmap and event/ui elements [exist](https://github.com/OpenVoiceOS/skill-ovos-setup/blob/d97365bdc4ae1f99afeaeb2f9502759b1f5c9a91/__init__.py#L584) but i don't know specifics

**Comment by JarbasAl** [comment]
it is there behind a setting in setup skill, we wanted to ensure we support minimum set of langs before exposing it

with whisper this is largely solved now as the main blocker was a offline option
